<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>任务列表</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_task"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView_task"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>习惯列表</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_habit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tableView_habit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>每日计划</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_plan"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_document">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_save"/>
   </widget>
   <widget class="QMenu" name="menu_theme">
    <property name="title">
     <string>主题</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_task">
    <property name="title">
     <string>任务</string>
    </property>
    <addaction name="action_addTask"/>
   </widget>
   <widget class="QMenu" name="menu_habit">
    <property name="title">
     <string>习惯</string>
    </property>
    <addaction name="action_addHabit"/>
   </widget>
   <widget class="QMenu" name="menu_plan">
    <property name="title">
     <string>计划</string>
    </property>
    <addaction name="action_deletePlan"/>
    <addaction name="action_addPlan"/>
   </widget>
   <addaction name="menu_document"/>
   <addaction name="menu_theme"/>
   <addaction name="menu_task"/>
   <addaction name="menu_habit"/>
   <addaction name="menu_plan"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_addTask">
   <property name="text">
    <string>新增任务</string>
   </property>
  </action>
  <action name="action_addHabit">
   <property name="text">
    <string>新增习惯</string>
   </property>
  </action>
  <action name="action_deletePlan">
   <property name="text">
    <string>删除计划</string>
   </property>
  </action>
  <action name="action_addPlan">
   <property name="text">
    <string>新增计划</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
